doc/policy/mempool-replacements.md

```diff
@@ -10,12 +10,7 @@ A transaction ("replacement transaction") may replace its directly conflicting t
 their in-mempool descendants (together, "original transactions") if, in addition to passing all
 other consensus and policy rules, each of the following conditions are met:
 
-1. If `-mempoolfullrbf=0` (the value is 1 by default), the directly conflicting transactions all signal replaceability explicitly. A transaction is
-   signaling BIP125 replaceability if any of its inputs have an nSequence number less than (0xffffffff - 1).
-   A transaction also signals replaceability if its version field is set to 3.
-
-   *Rationale*: See [BIP125
-   explanation](https://github.com/bitcoin/bips/blob/master/bip-0125.mediawiki#motivation).
+1. DEPRECATED: BIP125 signaling is no longer required.
```

Comment:
That's not what deprecated means, maybe just replace the whole thing with "(Removed)"